README.md

```diff
@@ -189,6 +192,7 @@ Where to discover new MATLAB resources.
 * [MATLAB Style Guide 1](http://note.sonots.com/Matlab/MatlabCodingStyle.html)
 * [MATLAB Style Guide 2](http://4dpiecharts.com/matlab-code-style-guide/)
 * [Octave and MATLAB Snippets](https://git.io/mtips) - Collection of code snippets
+* [knkutils](https://github.com/kendrickkay/knkutils) - MATLAB utility functions written by Kendrick Kay 
```

Comment:
I first thought that those functions were preposterous.
E.g. `vflatten` is literally just
```matlab
function [v] = vflatten(v)
v = v(:);
```
with documentation.

However, then I thought again, and saw the pure awesomeness this gives for [functional programming](http://blogs.mathworks.com/loren/2013/01/24/introduction-to-functional-programming-with-anonymous-functions-part-2/), e.g. the [FP constructs for MATLAB](http://www.mathworks.com/matlabcentral/fileexchange/39735-functional-programming-constructs).

Together with the sometimes overly rigid syntax of MATLAB (e.g. `sum(ones(3,1,3))(:)` is illegal), that toolbox is a hidden gem. Thank you :+1: